client/gutenberg/editor/components/header/more-menu/index.js

```diff
@@ -0,0 +1,63 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import { localize } from 'i18n-calypso';
```

Comment:
I'm just curious. What is the different between `import { localize } from 'i18n-calypso';` and `import { __ } from '@wordpress/i18n';`?

I can see both imports, not only on this PR but also on the current codebase, so no need to worry about that right now, but we might want to check if this somehow can affect the translations.

Reply:
AFAIK (but I might be wrong on this) they use different translations files: `@wordpress/i18n` use the Core WP ones, while `i18n-calypso` our owns.
In this particular case, the "Switch to classic editor" string is introduced by us, and therefore might not be available in Core.

Reply:
@akirk sorry I forgot what we came up with, I think `i18n-calypso` will eventually use the `@wordpress/i18n package` internally?